src/open_inwoner/openzaak/tests/test_zgw_imports_command.py

```diff
@@ -32,55 +31,77 @@ class ZGWImportTest(ClearCachesMixin, TestCase):
     def setUpTestData(cls):
         super().setUpTestData()
 
-        ZGWApiGroupConfigFactory(
-            ztc_service__api_root=CATALOGI_ROOT,
-            form_service=None,
-        )
+        cls.config = OpenZaakConfig.get_solo()
+        cls.roots = (CATALOGI_ROOT, ANOTHER_CATALOGI_ROOT)
+        cls.api_groups_for_root = {
+            root: ZGWApiGroupConfigFactory(ztc_service__api_root=root)
+            for root in cls.roots
+        }
 
-    @skip("Being fixed in the next PR")
     def test_zgw_import_data_command(self, m):
-        CatalogMockData().install_mocks(m)
-        InformationObjectTypeMockData().install_mocks(m)
+        m.reset_mock()
+        for root in self.roots:
+            CatalogMockData(root).install_mocks(m)
+            InformationObjectTypeMockData(root).install_mocks(m)
+            # ZaakTypeMockData(root).install_mocks(m)
+
+            # TODO: ADD CATALOGI like in iotypes
 
         # run it to import our data
         out = StringIO()
         call_command("zgw_import_data", stdout=out)
 
-        self.assertEqual(CatalogusConfig.objects.count(), 2)
-        self.assertEqual(ZaakTypeConfig.objects.count(), 2)
-        self.assertEqual(ZaakTypeInformatieObjectTypeConfig.objects.count(), 3)
-        self.assertEqual(ZaakTypeStatusTypeConfig.objects.count(), 2)
-        self.assertEqual(ZaakTypeResultaatTypeConfig.objects.count(), 2)
+        self.assertEqual(CatalogusConfig.objects.count(), 4)
+        self.assertEqual(ZaakTypeConfig.objects.count(), 4)
+        self.assertEqual(ZaakTypeInformatieObjectTypeConfig.objects.count(), 6)
+        self.assertEqual(ZaakTypeStatusTypeConfig.objects.count(), 4)
+        self.assertEqual(ZaakTypeResultaatTypeConfig.objects.count(), 4)
 
         stdout = out.getvalue().strip()
 
         expected = inspect.cleandoc(
             """
-        imported 2 new catalogus configs
-        aaaaa - 123456789
-        bbbbb - 123456789
+        imported 4 new catalogus configs
+        aaaaa - 123456789 [andere-catalogi.nl]
+        aaaaa - 123456789 [catalogi.nl]
+        bbbbb - 123456789 [andere-catalogi.nl]
+        bbbbb - 123456789 [catalogi.nl]
 
-        imported 2 new zaaktype configs
+        imported 4 new zaaktype configs
+        AAA - zaaktype-aaa
         AAA - zaaktype-aaa
         BBB - zaaktype-bbb
+        BBB - zaaktype-bbb
```

Comment:
Can we distinguish between these configs as in the case of the catalogi messages above (e.g. by displaying the netloc of the service api root or something else)?. It's quite useful in the catalogus case.